src/components/pipelines-overview/PipelinesOverviewPage.tsx

```diff
@@ -11,26 +11,35 @@ import NameSpaceDropdown from './NamespaceDropdown';
 import PipelineRunsListPage from './list-pages/PipelineRunsListPage';
 import TimeRangeDropdown from './TimeRangeDropdown';
 import RefreshDropdown from './RefreshDropdown';
+import { useActiveNamespace } from '@openshift-console/dynamic-plugin-sdk';
 
 interface PipelinesOverviewPageProps {
   match: Rmatch<never>;
 }
 
 const PipelinesOverviewPage: React.FC<PipelinesOverviewPageProps> = () => {
   const { t } = useTranslation('plugin__pipeline-console-plugin');
+  const [activeNamespace, setActiveNamespace] = useActiveNamespace();
+
```

Comment:
better to get the namespace from the match prop.

Reply:
Since we need `setActiveNamespace` to update the namespace, can we keep `activeNamespace` only? Anyhow we are already fetching from the hook `useActiveNamespace`

Reply:
advantage of getting the namespace from the match prop is that the user can use the dashboard in two different tabs for 2 different namespaces. And I don't think we need to maintain the state for the namespace get the namespace here and pass it to the children component.